### PR TITLE
NAS-127943 / 24.10 / Initial commit

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Section: admin
+Priority: optional
+Maintainer: Vladimir Vinogradenko <vladimirv@ixsystems.com>
+Build-Depends: debhelper-compat (= 12), wget
+Standards-Version: 4.4.0
+Homepage: https://github.com/restic/restic/releases
+
+Package: restic
+Architecture: amd64
+Description: restic is a backup program that is fast, efficient and secure
```
Comment (yocalebo) [inline debian/control:11]: Duplicate sentences in the `Description:`

File: debian/copyright [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: restic
+Source: https://github.com/restic/restic
+
+Files: *
+Copyright: 2020 iXsystems Inc
```
Comment (yocalebo) [inline debian/copyright]: Please remove references to BSD licenses. We're not altering this package in any way so we'll just  reference upstream copyright/license.

**Review by yocalebo — CHANGES_REQUESTED** [review]

File: debian/copyright (hunk 2 above)
Comment (themylogin) [reply debian/copyright]: Is there an example of how this file should look like?

Comment (yocalebo) [reply debian/copyright]: @themylogin Just leave the `Format:, Upstream-Name:` and `Source:` lines and remove everything else?

File: debian/control (hunk 1 above)
Comment (themylogin) [reply debian/control:11]: These are different descriptions (short and long description), the build will fail if I only leave the first line.

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Source: restic
+Section: admin
+Priority: optional
+Maintainer: Vladimir Vinogradenko <vladimirv@ixsystems.com>
```
Comment (yocalebo) [inline debian/control]:
```suggestion
Maintainer: iXsystems Development Team <dev@ixsystems.com>
```

**Review by yocalebo — APPROVED** [review]

**Pull request merged**